SecurityPkg/Library/Tcg2InitEventLib/Tcg2InitEventLib.c

```diff
@@ -0,0 +1,143 @@
+/** @file
+  Initializes TCG event data structures
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+#include <Library/Tpm2CommandLib.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseLib.h>
+#include <Library/UefiLib.h>
+#include <Library/UefiLib/UefiLibInternal.h>

```

Comment:
Library/UefiLib/UefiLibInternal.h should not be used here.  It is not a public header file